api/v1alpha2/conversion_test.go

```diff
@@ -105,11 +107,20 @@ func TestConvertMachine(t *testing.T) {
 						MachineClusterLabelName: "test-cluster",
 					},
 				},
+				Spec: MachineSpec{
```

Comment:
Can you adjust the name of this test case too please?

Reply:
Updated to `should convert all fields correctly`

Reply:
@ncdc unable to comment on the line so pasting link here. Should this test case name be changed too? https://github.com/kubernetes-sigs/cluster-api/pull/1938/files#diff-c8f1f9d0a7bf481d8e912ebc493d3c9fR35

Reply:
That's linking to a line that's not part of a test case, but we're good enough for now.